analyzers/pattern_detector.py

```
@@ -1,180 +1,152 @@
+                    "repeated_failures": repeated_failures,
+                },
+                "performance_patterns": {
+                    "slow_protocols": slow_protocols,
+                },
+                "usage_patterns": {
+                    "top_protocols": top_protocols,
+                },
+            }
 
-    async def _generate_insights(
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)